activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeBPMNTimerIT.java

```diff
@@ -247,13 +247,12 @@ public void shouldGetTimerCanceledEventOnBoundaryEvent() {
 
     @Test
     public void shouldExecuteProcessWithTimerStartExtension() {
-
-        securityUtil.logInAs("user");
+        securityUtil.logInAs("admin");
```

Comment:
are we sure about this change? I got that most probably we need to fetch the instance as an admin but won't be this a breaking change?

Reply:
yes because it's the process is started by a timer and has no initiator and per this PR only initiator or an admin can see a process instance

Reply:
yep we have a tests that verify an initiator can access his process instance and a non-inititiator can't access a process he didn't start.